### DISCUSSION TITLE
Best SIEM with Security Onion

### DISCUSSION THREAD

**Discussion by Sagaroth** [Security-Onion-Solutions/securityonion#4288] [Unsupported Versions] [answered]
Hi everyone!

I'm planning to deploy Security Onion in my work environment, where there are about 70 servers / network hardware.
I am rather new to Security Onion. And I would like to know what tips you would advise me and what SIEM I could couple with!

Thank you :)

**Comment by gcsrellis** [comment]
You could point your syslog/netflow/windows log servers towards Security Onion and have Kibana be your SIEM.  The Elastic Stack is a very powerful tool.

**Reply by Sagaroth** [reply]
Thanks for your answer!
I saw that it looked very complete indeed!
However, is there any use to put both the Wazuh agent and the Winlogbeats agent on my servers? Or would it cause a redundancy of the logs that would not be very useful?

**Reply by gcsrellis** [reply]
I think that would be environment dependent.  If you have another host based security tool, then no.  I use a windows event forwarder server with Winlogbeats and a GPO for my servers.  That way only one server is collecting all my logs and sending them to security onion instead of 100 servers sending logs individually.

**Comment by dougburks** [comment, marked as answer]
Many organizations find that the included tools like [Alerts](http://securityonion.net/docs/alerts), [Hunt](http://securityonion.net/docs/hunt), [Playbook](http://securityonion.net/docs/playbook), [TheHive](http://securityonion.net/docs/thehive), and [Kibana](http://securityonion.net/docs/kibana) give them everything they need.
